database.py

```diff
@@ -643,7 +643,7 @@ def createObsTables(self):
         TODO: Will need to provide better separation between Observers / Stations in future releases.
         """
         createquery = """CREATE TABLE IF NOT EXISTS Station (
-            station_num INT UNSIGNED NOT NULL,  /* 4 Digit COSPAR number of observation station */
+            station_num VARCHAR(4) NOT NULL,    /* 4 Digit COSPAR number of observation station */
```

Comment:
`unique not null` would be good in the future

Reply:
@Kmoneal You just need to add `UNIQUE ` to the beginning of this line.  No reason to defer it to the future, and this is why I provided it in the code review yesterday:
https://github.com/consensys-space/trusat-backend/blob/1eaaa0e16537c1ad103adc9a430e61b2b3751b36/database.py#L658

The changed line should be:
UNIQUE KEY Station_station_num_idx (station_num) USING BTREE,